doc/api/tracing.md

```diff
@@ -80,7 +80,7 @@ string that supports `${rotation}` and `${pid}`:
 node --trace-event-categories v8 --trace-event-file-pattern '${pid}-${rotation}.log' server.js
 ```
 
-Starting with Node.js 10.0.0, the tracing system uses the same time source
+Starting with Node.js v10.0.0, the tracing system uses the same time source
 as the one used by `process.hrtime()`
 however the trace-event timestamps are expressed in microseconds,
 unlike `process.hrtime()` which returns nanoseconds.
```

Comment:
The addition of `v` here violates our style guide as noted by @richardlau above. However, the text here could stand some other improvements:

* Might be a good idea to remove "Starting with Node.js 10.0.0". These docs are for a specific version of Node.js. 
* The sentence is a run-on sentence. It would be better split into two sentences.

```suggestion
The tracing system uses the same time source
as the one used by `process.hrtime()`.
However the trace-event timestamps are expressed in microseconds,
unlike `process.hrtime()` which returns nanoseconds.
```

Reply:
@Trott - thanks for the review, I understand. I have followed your suggestions. Please have a look.